ios/FluentUI/Vnext/Core/UIKit+SwiftUI_interoperability.swift

```diff
@@ -28,7 +28,9 @@ public struct UIViewAdapter: UIViewRepresentable {
             return
         }
 
-        stackView.removeAllSubviews()
+        for view in stackView.arrangedSubviews {
```

Comment:
is this related change?

Reply:
That's because we don't want to add the UIView+extensions.swift into the podspec.
I just changed not to use that extension and save that bit of space.